Cargo.toml

```diff
@@ -2,7 +2,140 @@
 members = ["node", "pallets/*", "primitives", "runtimes/*", "support"]
 resolver = "1"
 
+[workspace.package]
+version= "2.5.0"
+authors = ["Nodle Developers <eng@nodle.com>"]
+edition = "2021"
+
 [profile.release]
 # Substrate runtime requires unwind apparently, and anyways it gives more useful
 # panic messages which can be useful for troubleshooting purposes
 panic = "unwind"
+
+[workspace.dependencies]
+#Polkadot-SDK
+cumulus-client-cli = { version = "0.7.0", default-features = false }
```

Comment:
How do we know which crates versions of these packages are tested alongside each other for a particular upstream release?